lib/puma/server.rb

```diff
@@ -97,6 +98,10 @@ def tcp_mode!
       @mode = :tcp
     end
 
+    def early_hints!
```

Comment:
I think you were copying tcp_mode!, but that's a `!` method because it doesn't make sense for a server to be in tcp_mode and then *not* be in tcp mode later. Although I can't think of why anyone ever would (restarting and disabling early hints as the config changes?), it's more plausible that a server could toggle its support of early_hints. So this should probably just be treated like a straight-up accessor.

Reply:
Moved to an `attr_accessor` and deleted this method.